test/unit/manipulation.js

```diff
@@ -3099,3 +3099,43 @@ testIframe(
 		} );
 	}
 );
+
+QUnit.test( "should handle custom 'removeondestroy' event correctly", function( assert ) {
```

Comment:
This event is just an example, let's describe the issue here, with a reference to the issue number as well. How about:
```suggestion
QUnit.test( "should handle node removal in event's remove hook (gh-5214)", function( assert ) {
```

Reply:
Done